Sources/NetworkProtection/Networking/NetworkProtectionClient.swift

```diff
@@ -75,23 +75,24 @@ struct RedeemResponse: Decodable {
 public final class NetworkProtectionBackendClient: NetworkProtectionClient {
 
     enum Constants {
-        static let developmentEndpoint = URL(string: "https://staging.netp.duckduckgo.com")!
+        static let productionEndpoint = URL(string: "https://controller.netp.duckduckgo.com")!
+        static let stagingEndpoint = URL(string: "https://staging.netp.duckduckgo.com")!
```

Comment:
This could probably be removed, it's not used any more - kept it for the time being in case we want to add a switcher, but it would be better to offer a free-form text field to set a custom endpoint.

Reply:
I'm ok with keeping it for now, and agree on the switcher comment.